charts/dealer/templates/dealer-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "dealer.fullname" . }}
+  labels:
+    app: {{ template "dealer.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "dealer.fullname" . }}
+      release: {{ .Release.Name }}
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "dealer.fullname" . }}
+        release: "{{ .Release.Name }}"
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "3333"
+        prometheus.io/scrape: "true"
+      {{- with .Values.annotations }}
+        {{ toYaml . | nindent 8 }}
+      {{- end }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}@{{ .Values.image.digest }}"
+        env:
+        - name: NETWORK
+          value: regtest
+        - name: LOG_LEVEL
+          value: debug
```

Comment:
Why debug by default?

Reply:
These were the defaults set in [.envrc](https://github.com/GaloyMoney/dealer/blob/main/.envrc)

Reply:
the log level was "debug" for the local/dev setup or at least I set it so in the sample .envrc file
we can use "info" or whatever the default in the chart